### PR TITLE
Alumni 2021

### PR DESCRIPTION
**Pull request by gonuke**
Move folks to alumni status for Spring 2021

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -114,6 +106,9 @@
     role: Project Assistant
     employer: 
     employer_url:
+  - name: "Anna Caldwell-Overdier"
```
[the remark this answers is not in the document]
Comment (gonuke) [reply _data/people.yml:109]: Probably not, then.  I copied it from its previous location... will update

File: _data/people.yml [hunk 2]
```diff
@@ -238,6 +236,12 @@
     role: MS
     employer: 
     employer_url: 
+  - name: Erik Nygaard
+    url: 
+    year: 2011
+    Role: MS
```
Comment (kkiesling) [inline _data/people.yml:242]: `role` is not appearing when built. I think it needs to be lowercase.
nygaard

File: _layouts/people.html [hunk 3]
```diff
@@ -41,9 +41,9 @@ <h2>{{ group.group }}</h2>
           {% else %}
           {{ person.name }},
           {% endif %}
-          ({{ person.role }}, {{ person.year }}), 
+          ({{ person.role }}, {{ person.year }})
           {% if person.employer_url != null %}
-          <a href="{{ person.employer_url }}" target="_blank">{{ person.employer }}</a>
+          , <a href="{{ person.employer_url }}" target="_blank">{{ person.employer }}</a>
```
Comment (kkiesling) [inline _layouts/people.html:46]: FYI- with this change, there is no comma in front of the company name if there is no url (see example of the two different behaviors)
commas

**Review by kkiesling — CHANGES_REQUESTED** [review]
minor remaining comments now that I got to build this.

**Review by kkiesling — APPROVED** [review]
sorry, I forgot about this. lgtm.

**Pull request merged**